test/cmdlineTests.sh

```diff
@@ -306,6 +305,22 @@ SOLTMPDIR=$(mktemp -d)
     REPO_ROOT=$(pwd) # make it absolute
     cd "$SOLTMPDIR"
     "$REPO_ROOT"/scripts/isolate_tests.py "$REPO_ROOT"/docs/ docs
+
+    if node -v >/dev/null
+    then
+        if npm list -g | grep solhint >/dev/null
+        then
+            echo "node is installed, setting up solhint"
+            cp "$REPO_ROOT"/test/.solhint.json "$SOLTMPDIR"/.solhint.json
+        else
+            echo "node is installed, installing solhint"
+            npm install -g solhint
```

Comment:
I'm a bit uneasy with installing software globally in a test script. Perhaps it would be better to just install it in the circle setup routine and just don't run it if it is not available?

Reply:
I am too. Right now I am mostly still trying to figure out the best way to implement this, and I can't experiment without pushing to Ci, which is slow… The biggest stumbling block so far has actually been installing node itself, but getting there.